NetSuiteProject/TestFile.js

```diff
@@ -15,4 +15,6 @@ division("Zero", "Sub");
 division(10, 2);
 
 remainder(4,1);
-remainder(10, 2);
\ No newline at end of file
+remainder(10, 2);
+remainder("Hello", "World");
```

Comment:
This is a *great* idea to build test cases that cover invalid inputs for your functionality. Can you think of other types of invalid inputs besides strings? Shouldn't you test your other functions for similar invalid inputs as well?